java/server/src/org/openqa/grid/internal/utils/SelfRegisteringRemote.java

```diff
@@ -284,17 +284,17 @@ private void registerToHub(boolean checkPresenceFirst) {
           registrationRequest.getConfiguration().browserTimeout = hubConfiguration.browserTimeout;
         }
 
-        LOG.fine("Updated node configuration: " + new Json()
+        LOG.severe("Updated node configuration: " + new Json()
             .toJson(registrationRequest.getConfiguration()));
       } catch (Exception e) {
-        LOG.warning(
+        LOG.severe(
             "Error getting the parameters from the hub. The node may end up with wrong timeouts." +
             e.getMessage());
       }
 
       try {
         URL registration = new URL(tmp);
-        LOG.info("Registering the node to the hub: " + registration);
+        LOG.severe("Registering the node to the hub: " + registration);
```

Comment:
Seems like too much of a jump. Remember, we are thinking that there is a certain *class* of logs for which this assumption will work. Perhaps we need to categorize the log level transformations. For example, transformations either take place from FINEST to/from INFO or WARNING to/from SEVERE.

Reply:
Maybe. The problem is that what is the standard to categorize the log level transformations. 

We have a pending issue to process a jump: https://github.com/ponder-lab/Logging-Level-Evolution-Plugin/issues/208. I am not sure whether we need to do it. 

I am also thinking that how to improve the boundary to avoid jump. By looking at the figure for log level distributions in the 4 projects below (it is from this paper https://dl.acm.org/citation.cfm?id=3128785.3128834 ), the number of debug, info and error is always greater than the number of rest two log levels. If we would like build boundary for this logging framework, can we give a greater interval to debug, info and error? i.e., can we build the boundary based on log distribution in the real project? Then, we could reduce drastic log transformations.

![image](https://user-images.githubusercontent.com/10117031/65782709-45ecee00-e11c-11e9-8ae4-e7416bf09de0.png)